src/front-end/src/components/Footer/index.tsx

```diff
@@ -48,7 +49,7 @@ const Footer = () => {
     <footer>
       <div className="footer">
         <div className="left">
-          <Link id="arad-codeLink" to="/code">
+          <Link id="arad-codeLink" to={location.pathname !== "/search" ? "/code" : "/"}>
```

Comment:
So this will change the link in the lower right, correct? Meaning that if the path is /search it will go to the code, but if ti's not it will redirect home? Why are we doing this? Can't we alway find home through another link?

Reply:
I did it so I could use the arad title (bottom left) as a link back to home, in this case to exit the search and return. Yes I can totally remove it if we want a different nav button for home.